crates/typst-library/src/math/frac.rs

```diff
@@ -93,7 +102,15 @@ fn layout(
     ctx.unstyle();
 
     ctx.style(ctx.style.for_denominator());
-    let denom = ctx.layout_frame(denom)?;
+    let denom = ctx.layout_frame(&Content::sequence(
+        denom
+            .iter()
+            // add a comma between each element
+            .zip(iter::repeat(TextElem::packed(",")))
```

Comment:
this looks like the result will be tight without any space after the comma. is that desirable?

Reply:
I added spaces before committing but no matter how many spaces I added after the comma, none showed up after compilation. I think they are removed because if I add some other character after many spaces, the spaces will be rendered. I agree that one space would be nice, any suggestions as to how?

Reply:
Nevermind, it's alright as is! Math mode automatically adds appropriate spacing after commas. I forgot about that.